`Data.IntMap.CharMap2` uses deprecated `Data.IntMap.showTree`

Compiler warning: `Data.IntMap.CharMap2` uses deprecated `Data.IntMap.showTree`:
```
Building library for regex-tdfa-1.3.1.0..
[ 1 of 25] Compiling Data.IntMap.CharMap2 ( lib/Data/IntMap/CharMap2.hs, dist/build/Data/IntMap/CharMap2.o )

lib/Data/IntMap/CharMap2.hs:258:24: warning: [-Wdeprecations]
    In the use of ‘showTree’
    (imported from Data.IntMap, but defined in containers-0.5.11.0:Data.IntMap.Internal.DeprecatedDebug):
    Deprecated: "These debugging functions will be removed from this module. They are available from Data.IntMap.Internal.Debug."
    |
258 | showTree (CharMap m) = MD.showTree m
    |                        ^^^^^^^^^^^
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,7 +4,7 @@ For the package version policy (PVP), see  http://pvp.haskell.org/faq .
 
 _2022-07-14, Andreas Abel_
 
-- Fix an `undefined` in `Show PatternSet` [#37](https://github.com/haskell-hvr/regex-tdfa/issues/37))
+- Fix an `undefined` in `Show PatternSet` ([#37](https://github.com/haskell-hvr/regex-tdfa/issues/37))
 - Document POSIX character classes (e.g. `[[:digit:]]`)
 
 ### 1.3.1.2 Revision 1

--- a/lib/Data/IntMap/CharMap2.hs
+++ b/lib/Data/IntMap/CharMap2.hs
@@ -9,7 +9,7 @@ import Data.Char (chr)
 import Data.Char as C(ord)
 import Data.List as L (map)
 import qualified Data.IntMap as M
-#if MIN_VERSION_containers(0,6,0)
+#if MIN_VERSION_containers(0,5,11)
 import qualified Data.IntMap.Internal.Debug as MD
 #else
 import qualified Data.IntMap as MD

--- a/lib/Data/IntMap/EnumMap2.hs
+++ b/lib/Data/IntMap/EnumMap2.hs
@@ -4,7 +4,7 @@ module Data.IntMap.EnumMap2 where
 
 import Data.Foldable as F (Foldable(foldMap))
 import qualified Data.IntMap as M
-#if MIN_VERSION_containers(0,6,0)
+#if MIN_VERSION_containers(0,5,11)
 import qualified Data.IntMap.Internal.Debug as MD
 #else
 import qualified Data.IntMap as MD

